pkg/image/api/validation/validation_test.go

```diff
@@ -315,24 +315,6 @@ func TestValidateImageStreamMappingNotOK(t *testing.T) {
 			field.ErrorTypeRequired,
 			"image.metadata.name",
 		},
-		"invalid repository pull spec": {
```

Comment:
Ok, so at least change the repository to something like `///a/b/c/d` to generate the `ErrorTypeInvalid`.